bin/embedding-sdk/fixup-types-after-compilation.js

```diff
@@ -42,24 +41,30 @@ const traverseFilesTree = dir => {
   }
 };
 
+const getRelativePath = (fromPath, toPath) => {
+  const relativePath = path.relative(path.dirname(fromPath), toPath);
+  return relativePath.startsWith(".") ? relativePath : "./" + relativePath;
```

Comment:
This logic doesn't seem to always work.

```
export { FlexibleSizeComponent } from ".//components/private/util/FlexibleSizeComponent";
```

I mean that works, but I wouldn't want to use double slash like that.

Reply:
If i open `resources/embedding-sdk/dist/enterprise/frontend/src/embedding-sdk/components/public/index.d.ts`
I see 
<img width="705" alt="image" src="https://github.com/user-attachments/assets/11a4753d-9b5a-4f81-ae67-2cafa7bdddf1">

`FlexibleSizeComponent` hasn't been in the private folder for a few days I think

Reply:
🤔 Do I get the old type? I only run `yarn build-embedding-sdk` and install it in my setup. Maybe I need to test everything again.

Reply:
I checked the package on resources directly to avoid caching issues. I do have the feeling that sometimes the build it's messed up and i need to nuke the sdk folder in resources/